src/synapse/README.md

```diff
@@ -0,0 +1,98 @@
+# Microsoft Azure CLI Synapse Extension #
+This is a extension for Synapse features.
+
+### How to use ###
+
+Install this extension using the below CLI command
+
+```bash
+az extension add --name synapse
+```
+
+### Included Features
+
+#### Workspace
+
+Manage Synapse Workspaces.
+
+*Examples:*
+
+```bash
+az synapse workspace create \
+    --name fromcli4 \
+    --resource-group rg \
+    --account-url https://testadlsgen2.dfs.core.windows.net \
+    --file-system testfilesystem \
+    --sql-admin-login-user cliuser1 \
+    --sql-admin-login-password Password123! \
+    --location "East US"
+```
+
+#### Spark Pool

```

Comment:
Also add example for SQL Pool?

Reply:
added

---
In reply to: [386411165](https://github.com/Azure/azure-cli-extensions/pull/1339#discussion_r386411165) [](ancestors = 386411165)